.changeset/all-loops-deny.md

```diff
@@ -0,0 +1,9 @@
+---
+'astro': minor
+---
+
+If you're an adapter developer, and use the `NodeApp` for your adapter, the function  `NodeApp.match` method to accept a second, optional parameter.
```

Comment:
```suggestion
Updates the `NodeApp.match()` function in the Adapter API to accept a second, optional parameter to allow adapter authors to add headers to static, prerendered pages.
```
Can we maybe start with a use case for this? (I made one up, so make it accurate!) 

I think maybe the structure above makes it still clear that it's for adapter developers, and keeps the same amount of technical detail, but relating this to the general use case is still helpful!